Content/src/Client/Client.fs

```diff
@@ -151,15 +178,29 @@ let update (msg : Msg) (currentModel : Model) : Model =
 let update (msg : Msg) (currentModel : Model) : Model * Cmd<Msg> =
     match currentModel.Counter, msg with
     | Some counter, Increment ->
+        #if (bridge)
+        let nextCounter = { counter with Value = counter.Value + 1 }
+        currentModel, Cmd.bridgeSendOr ServerMsg.Increment (Remote(SyncCounter nextCounter))
+        #else
         let nextModel = { currentModel with Counter = Some { Value = counter.Value + 1 } }
         nextModel, Cmd.none
+        #endif
     | Some counter, Decrement ->
+        #if (bridge)
+        let nextCounter = { counter with Value = counter.Value - 1 }
+        currentModel, Cmd.bridgeSendOr ServerMsg.Decrement (Remote(SyncCounter nextCounter))
+        #else
         let nextModel = { currentModel with Counter = Some { Value = counter.Value - 1 } }
         nextModel, Cmd.none
+        #endif
+    #if (bridge)
+    | _, Remote(SyncCounter counter) ->
+        { currentModel with Counter = Some counter}, Cmd.none
+    #else
     | _, InitialCountLoaded initialCount->
         let nextModel = { Counter = Some initialCount }
         nextModel, Cmd.none
-
+    #endif
```

Comment:
don't forget to add your credits to `safeComponents` !

Reply:
Oops 😅